index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+    <html lang="en">
+        <head>
+            <meta charset="UTF-8">
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <link rel="stylesheet" href="style.css">
+            <title>Lições aprendidas</title>
+        </head>
+
+        <body>
+            <header class="cabecalho" id="cabecalho">
+                <h1 id="titulo">Lições aprendidas</h1>
+                <nav>
+                    <a href="https://www.linkedin.com/in/mariana-saraiva-977a251b3/" target="_blank">LindedIn</a>
+                    <a href="https://github.com/marianasaraiva" target="_blank">Github</a>
+                </nav>
+            </header>
+
+            <main class="main">
+                <section class="section lado-direito">
+                    <h2>Lições aprendidas nos últimos dias</h2>
+                    <nav>
+                        <ol id="licoes_aprendidas">
+                            <li>Respeitar o seu tempo de aprendizagem.</li>
+                            <li>Aprender a pescar, ou seja, buscar a metodologia ativa de aprendizado.</li>
+                            <li>Que sozinhos podemos ir mais rápidos, porém juntos vamos mais longe.</li>
+                            <li>Ensinar é muito prazeroso e é a chave so sucesso.</li>
+                            <li>Aprender a importância do autoconhecimento para gerenciar sua carreira.</li>
+                            <li>Disciplina e foco são aracterísticas que podem ser aprimoradas.</li>
+                            <li>Buscar conhecimento contínuo é um caminho sem volta.</li>
+                            <li>Pedir ajuda é libertador após se empenhar na busca por respostas, mas mesmo assim não encontrar.</li>
+                            <li>Melhorar a capacidade de escrita, especialmente a ecessidade de síntase.</li>
```

Comment:
@marianasaraiva acredito que tenha um erro tipográfico neste item.

```suggestion
                            <li>Melhorar a capacidade de escrita, especialmente a necessidade de síntase.</li>
```